pkg/api/openshiftcluster.go

```diff
@@ -247,6 +247,15 @@ const (
 	MTU3900 MTUSize = 3900
 )
 
+// OutboundType represents the type of routing a cluster is using
+type OutboundType string
+
+// OutboundType constants
+const (
+	OutboundTypeUserDefinedRouting OutboundType = "UserDefinedRouting"
+	OutboundTypeLoadbalancer       OutboundType = "Loadbalancer"
```

Comment:
Should this be `LoadBalancer`?

Reply:
This should probably get updated just to match upstream, though I will point out ARO-RP uses the same value https://github.com/Azure/ARO-RP/blob/master/pkg/api/openshiftcluster.go#L257.  Should I go ahead and update both?

Reply:
Yes please